### PR TITLE
Update faq.rst about missing task logs

### PR DESCRIPTION
**Pull request by RNHTTR**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
I like the description and good idea to mention it, maybe others would like to add something?

File: docs/apache-airflow/faq.rst [hunk 1]
```diff
@@ -441,6 +441,17 @@ If the tasks are not related by dependency, you will need to :ref:`build a custo
 Airflow UI
 ^^^^^^^^^^
 
+Why did my task fail with no logs in the UI?
+--------------------------------------------
```
Comment (jedcunningham) [inline docs/apache-airflow/faq.rst:445]:
```suggestion
--------------------------------------------

```

nit

File: docs/apache-airflow/faq.rst [hunk 2]
```diff
@@ -441,6 +441,17 @@ If the tasks are not related by dependency, you will need to :ref:`build a custo
...
+Logs are [typically served when a task reaches a terminal state](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/logging-tasks.html#serving-logs-from-workers). Sometimes, a task's normal lifecycle is disrupted, and the task's
```
Comment (jedcunningham) [inline docs/apache-airflow/faq.rst]: Can you switch this link to be a ref, vs an external link?

File: docs/apache-airflow/faq.rst [hunk 3]
```diff
@@ -441,6 +441,17 @@ If the tasks are not related by dependency, you will need to :ref:`build a custo
...
+worker is unable to write the task's logs. This typically happens for one of two reasons:
+
+1. [Zombie Tasks](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks)
+2. Tasks failed after getting stuck in queued (Airflow 2.6.0+)
+
+Setting retries for each task drastically reduces the chance that either of these problems impact a workflow.
+
```
Comment (jedcunningham) [inline docs/apache-airflow/faq.rst:454]:
```suggestion
```

nit

File: docs/apache-airflow/faq.rst [hunk 4]
```diff
@@ -441,6 +441,17 @@ If the tasks are not related by dependency, you will need to :ref:`build a custo
...
 ^^^^^^^^^^
 
+Why did my task fail with no logs in the UI?
+--------------------------------------------
+
+Logs are [typically served when a task reaches a terminal state](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/logging-tasks.html#serving-logs-from-workers). Sometimes, a task's normal lifecycle is disrupted, and the task's
+worker is unable to write the task's logs. This typically happens for one of two reasons:
+
+1. [Zombie Tasks](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks)
+2. Tasks failed after getting stuck in queued (Airflow 2.6.0+)
```
Comment (eladkal) [inline docs/apache-airflow/faq.rst]: What the Airflow 2.6.0+ means?
if task is stuck in queue it means it didn't reach worker. "extracting" it from the queue can be done with any Airflow version.

Comment (RNHTTR) [reply docs/apache-airflow/faq.rst]: #30375 will fail tasks that get stuck in queued for some reason. This will result in missing task logs

Comment (eladkal) [reply docs/apache-airflow/faq.rst]: So we should refer to the relevant Airflow setting task_queued_timeout and ask user to check if the threshold chosen is OK for them

**Review by potiuk — APPROVED** [review]

**Pull request merged**